src/mongo/base/error_codes.err

```diff
@@ -117,6 +117,7 @@ error_code("InitialSyncOplogSourceMissing", 114)
 error_code("CommandNotSupported", 115)
 error_code("DocTooLargeForCapped", 116)
 error_code("ConflictingOperationInProgress", 117)
+error_code("FailedToSatisfyReadPreference", 133)
```

Comment:
Because it came in here through a cherry-pick. I don't actually know what the 'backported' section is about, really.